src/redux/reducers/profileReducer.test.js

```diff
@@ -0,0 +1,38 @@
+import profileReducer from './profileReducer';
```

Comment:
Consider following the import grouping principle.

Reply:
+